.circleci/config.yml

```diff
@@ -31,7 +31,7 @@ jobs:
           command: hokusai configure --kubectl-version 1.6.3 --s3-bucket artsy-citadel --s3-key k8s/config --platform linux
       - run:
           name: Deploy
-          command: hokusai staging deploy $CIRCLE_SHA1
+          command: hokusai staging deploy $CIRCLE_SHA1 --git-remote origin
```

Comment:
Do we want `origin` here or do we want `upstream`/`artsy`?

Reply:
It's running on CI, so it should be origin